apis/frontend/react-support/README.md

```diff
@@ -1,3 +1,279 @@
 # Nosto React
 
```

Comment:
follow the same naming convention for folders and use `nosto-react` as folder name?

Reply:
that would be good, but since this pre-existing, I don't want to break existing links